package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "connection-manager-ui",
-  "version": "1.6.11",
+  "version": "1.6.12",
```

Comment:
Feels unimportant, but this is probably a minor feature? I dunno. Debatable I suppose.